### PR TITLE
Hero video, clean up old JS

### PR DESCRIPTION
**Pull request by craigcook**
This makes the hero videos play in a modal on the home page and diversity page. It also removes some redundant JS from the base template.

### CODE REVIEW

File: bedrock/careers/templates/careers/includes/big-picture-video.html [hunk 1]
```diff
@@ -4,10 +4,14 @@
...
 #}
 
-<div class="mzp-c-video">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/_obdx3VUzuQ" frameborder="0" allow="encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
-  <p><a href="https://www.youtube.com/watch?v=_obdx3VUzuQ">{{ ftl('ui-watch-the-video') }}</a>
-  </p>
-  </iframe>
-</div>
+<div class="c-hero-video">
+  <a class="video-play js-video-play" href="https://youtu.be/F-nFQryDB0s" title="Play the video">
```
Comment (reemhamz) [inline bedrock/careers/templates/careers/includes/big-picture-video.html]: It looks like you embedded the wrong video here, it should be The Big Picture: https://www.youtube.com/watch?v=_obdx3VUzuQ

File: bedrock/careers/templates/careers/diversity.html [hunk 2]
```diff
@@ -20,11 +20,10 @@
         block_class='mzp-l-split-center-on-sm-md',
         media_class='mzp-l-split-h-center',
         media_include='careers/includes/diversity-video.html',
-        image_url='img/careers/diversity-video-thumbnail.png',
         media_after=True
         ) %}
-        <h1>Diversity & inclusion at Mozilla</h1>
-        <a href="{{ url('mozorg.diversity.2021.index') }}" class="mzp-c-button mzp-t-product">View our 2021 D&I Report</a>
+          <h1>Diversity & inclusion at Mozilla</h1>
+          <p><a href="{{ url('mozorg.diversity.2021.index') }}" class="mzp-c-button mzp-t-product">View our 2021 D&I Report</a></p>
```
Comment (reemhamz) [inline bedrock/careers/templates/careers/diversity.html:26]: **Question:** Why wrap the CTA in a `<p>` tag here?

File: bedrock/careers/templates/careers/home.html [hunk 3]
```diff
@@ -64,11 +64,10 @@
         block_class='mzp-l-split-center-on-sm-md',
         media_class='mzp-l-split-h-center',
         media_include='careers/includes/big-picture-video.html',
-        image_url='img/careers/careers-video-thumbnail.webp',
         media_after=True
         ) %}
-        <h1>Good work that does you good</h1>
-        <a href="{{ url('careers.listings') }}" class="mzp-c-button mzp-t-product">Find your role at Mozilla</a>
+          <h1>Good work that does you good</h1>
+          <p><a href="{{ url('careers.listings') }}" class="mzp-c-button mzp-t-product">Find your role at Mozilla</a></p>
```
Comment (reemhamz) [inline bedrock/careers/templates/careers/home.html:28]: **Question:** Same here with the `<p>` tag

**Review by reemhamz — APPROVED** [review]
The modal functionality works as planned. Thanks for that!
Just a couple of questions and one blocking nit (wrong video for homepage). I think that should be a quick fix-up. r+wc 🌺

File: bedrock/careers/templates/careers/includes/big-picture-video.html (hunk 1 above)
Comment (craigcook) [reply bedrock/careers/templates/careers/includes/big-picture-video.html]: 🤦

File: bedrock/careers/templates/careers/diversity.html (hunk 2 above)
Comment (craigcook) [reply bedrock/careers/templates/careers/diversity.html:26]: It just adds a little more semantic structure, helps the CTA stand on its own as a complete thought.

**Review by reemhamz — APPROVED** [review]
r+ 💅🏼

**Pull request merged**